### PR TITLE
Feat/fund3

### PR DESCRIPTION
**Pull request by AVtheking**
What kind of change does this PR introduce?
Feature for creating fundraising campaigns pledges for the user

Issue Number:

Fixes https://github.com/PalisadoesFoundation/talawa-api/issues/1663

Did you add tests for your changes?

Yes

Summary
This is a follow up PR for the issue https://github.com/PalisadoesFoundation/talawa-api/issues/1663  and PR  #1880 added remaining mutation and query for the campaing model and created mutation and queries for pledges

Does this PR introduce a breaking change?

No

### CODE REVIEW

**Comment by palisadoes** [comment]
Is this ready for review and does it cover the scenarios we discussed in the issue?

**Comment by AVtheking** [comment]
Yes sir

File: schema.graphql [hunk 1]
```diff
@@ -646,7 +646,7 @@ enum Frequency {
 
 type Fund {
   _id: ID!
-  campaign: [FundraisingCampaign]!
+  campaigns: [FundraisingCampaign]!
```
Comment (palisadoes) [inline schema.graphql:649]: Funds may or may not have associated campaigns. Please fix and adjust your tests.

**Review by palisadoes — CHANGES_REQUESTED** [review]

File: schema.graphql (hunk 1 above)
Comment (AVtheking) [reply schema.graphql]: fixed it .

**Review by palisadoes — APPROVED** [review]

**Pull request merged**